src/storage/worditem-remotedb-driver.js

```diff
@@ -216,4 +220,39 @@ export default class WordItemRemoteDbDriver {
   getCheckArray (dataItems) {
     return dataItems.map(item => this._makeStorageID(item))
   }
+
+  isTheSame (remoteItem, localItem) {
+    return this._makeStorageID(remoteItem) === this._makeStorageID(localItem)
+  }
+
+  comparePartly (changeItem, sourceItem) {
```

Comment:
see comments on these methods in indexeddbdriver - they apply here as well

Reply:
This method was removed